kube/services/metadata/metadata-deploy.yaml

```diff
@@ -49,6 +49,8 @@ spec:
         env:
           - name: GEN3_DEBUG
             GEN3_DEBUG_FLAG|-value: "False"-|
+          - name: REDIS_DB_HOST
```

Comment:
Until we need greater abstraction, I feel like this is a simple mechanism for pointing the MDS at redis.

@ZakirG this would invalidate the need for the `metadata.env` secret configuration update I previously suggested to you.